src/Components/Admin/Admin.tsx

```diff
@@ -65,6 +70,7 @@ const Admin = () => {
         const result = await deleteRooms(id);
         if (result.status == 200) {
             setLoaded(true);
+            handleClose(setShowDeleteModal);
```

Comment:
Copy pasted from https://github.com/robert-szalkai/meeting-rooms-booking/pull/9 . 
Don't pass the setter/dispatcher function here if you already have access to it in the scope of the component. You can bind handleClose to the prop: onClose={handleClose} and use the setShowModal in the function body